contour/src/test/kotlin/com/squareup/contour/ContourTests.kt

```diff
@@ -293,6 +293,19 @@ class ContourTests {
     assertThat(view.width).isEqualTo(33) // 260 * 0.13 = 33.8 ~= 33 floored
   }
 
+  @Test
+  fun `conversion of XFloat to XInt and YFloat to YInt`() {
+    val view = View(activity)
+    contourLayout(activity, width = 260, height = 260) {
+      view.layoutBy(
+          leftTo { parent.left() }.widthOf { (parent.width() * 0.42f).toInt() },
+          topTo { parent.top() }.heightOf { (parent.height() * 0.42f).toInt() }
+      )
+    }
+    assertThat(view.width).isEqualTo(109)   // 260 * 0.42 = 109.2 ~= 109 floored
+    assertThat(view.height).isEqualTo(109)
+  }
+
```

Comment:
@JakeWharton does this look good enough?

Reply:
Looks great! Just want to make sure if I change the method body to always return 42 that something will fail.